um_canonicalize/tags.yaml

```diff
@@ -115,6 +139,7 @@ categories:
     - EQT
   Definiteness:
     - DEF
+    - NDEF
```

Comment:
is this just a typo for `INDF`?

Reply:
Yeah, and we need to be cautious about introducing spurious tags into this script. (The data for 9 languages was wrong by including this tag. This is fixed locally.)